apps/patient/src/utils/general.ts

```diff
@@ -151,7 +151,7 @@ export const preparePharmacyHours = (pharmacy: types.Pharmacy): EnrichedPharmacy
   };
 };
 
-export const convertReadyByToUTCTimestamp = (readyBy: string): string => {
+export const convertReadyByToUTCTimestamp = (readyBy: string, readyByDay: string): string => {
```

Comment:
could you make a todo comment to make these params typed? so something like `if (readyByDay === 'tomorrow') {` is more type safe